### PR TITLE
ci: add `Question` issue template and improve ci about invalid question.

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

ci

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.

新增 `Question` 的 issue 模版并优化关于无效 issue 的 ci 配置。

#### (Optional) More detail description for this PR(en: English/zh: Chinese).

en: Added `Question` issue template, changed the cron duration for closing invalid issues to 1 hour, and added judgment on invalid issues related to Question.

zh(optional): 新增`Question` 的 issue 模板，关闭无效 issue 的 cron 时长改为1小时，并增加 Question 相关无效 issue 的判断。

#### Which issue(s) this PR fixes:

None.

### CODE REVIEW

**Review by li-jin-gou — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Question
+about: Ask a question, so we can help you easily
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```
[the remark this answers is not in the document]
Comment (L2ncE) [reply .github/ISSUE_TEMPLATE/question.md:9]: ok, I will modify it later, thx.

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ assignees: ''
 
 A clear and concise description of what the question is.
 
+**Reproducible Code**
+
+Please submit a reproducible code and tell us how to reproduce it like how he send a request or send what request?
```
Comment (Duslia) [inline .github/ISSUE_TEMPLATE/question.md:16]: how you send

Comment (L2ncE) [reply .github/ISSUE_TEMPLATE/question.md]: > how you send

done, thx.

Comment (Duslia) [reply .github/ISSUE_TEMPLATE/question.md]: Please construct a minimum complete and reproducible example for us to get the same error. And tell us how to reproduce it like how you send a request or send what request

Comment (L2ncE) [reply .github/ISSUE_TEMPLATE/question.md]: done

**Review by Duslia — APPROVED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**